src/lib/services/contentful/graphqlClient.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, expect, it, vi, type Mock } from "vitest";
+import getClient, { type Client } from "./graphqlClient";
+
+const query = "{ some { graphql { query } } }";
+
+const sampleData = Symbol("sample return value");
```

Comment:
Just curious, why do we use `Symbol` here?

Reply:
It was so that we could compare object identity while still having a value that's immediately understandable in console logs. We could use `{}` for the same effect, but when logged to the console it wouldn't immediately be obvious what that value represented.